lib/portlayer/storage/image_cache.go

```diff
@@ -300,3 +298,54 @@ func (c *NameLookupCache) ListImages(ctx context.Context, store *url.URL, IDs []
 
 	return imageList, nil
 }
+
+// DeleteImage deletes an image from the image store.  If it is in use or is being inheritted from, then this will return an error.
+func (c *NameLookupCache) DeleteImage(ctx context.Context, image *Image) error {
+	infof("DeleteImage: deleting %s", image.Self())
```

Comment:
Do you mean in addition to the `infof` above or in place of?  

I can add trace calls in a later PR.  You'll notice it's not being used anywhere in this package.